java/src/client/game/GameManager.java

```diff
@@ -60,22 +82,24 @@ public void stopPoller() {
      * When the poller finds out the server has a new version it will call this function to update the client game
      * @param cm the new clientModel
      */
-    public void updateClientModels(ClientModel cm){
+    public void updateGameManager(ClientModel cm){
         mClientModel = cm;
+        mFacadeManager.update(cm);
     }
 
 
     /**
      * Grabs a "server" for us so we can talk with the real server
      * @return a server we can use
      */
+
```

Comment:
while technically correct, this is a borderline style choice, no meaning is lost, and arguably a worthless fix this early in development.